doc/project/dev_env_and_tools.rst

```diff
@@ -378,7 +388,7 @@ in :ref:`stable_api_changes`.
 
 * ``Bug``
 
-The issue is a bug, or the PR is fixing a bug       |
```

Comment:
This typo was introduced in this pull request and therefore the fix should be squashed into the previous commit instead.

Reply:
Fixed in new version of this PR.